src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -611,8 +611,10 @@ public void tickCalcsAndLegacy(VehicleUpdateEvent event) {
 					vel.setY(5);
 				}
 				UEntityMeta.removeMetadata(car, "car.jumpFull");
+			} else if(UEntityMeta.hasMetadata(car, "car.frozen") || car.hasMetadata("car.frozen") && car.hasMetadata("kart.immune")) {
```

Comment:
Is this needed? Surely if a car is frozen it should set the y velocity to 0 still, not leave it untouched? (Or it wouldn't be frozen?).  And if it is needed please add parenthesis for clarity